### PR TITLE
Add 1709 Swarm Dockerfile

### PR DESCRIPTION
**Pull request by gamma**
Not sure if the build files should be updated as well ...

### CODE REVIEW

File: swarm/Dockerfile.1709 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+WORKDIR /go/src/github.com/docker/swarm
+RUN git checkout $env:SWARM_VERSION
+RUN go install .
+
+FROM microsoft/nanoserver:1709
+
+COPY --from=build /go/src/github.com/docker/swarm/swarm.exe /swarm.exe
+
+# nanoserver 1709 is missing the netapi32.dll
+COPY --from=build /Windows/System32/netapi32.dll /Windows/System32/netapi32.dll
```
Comment (StefanScherer) [inline swarm/Dockerfile.1709]: Oh, swarm.exe also needs this netapi32.dll?

Comment (gamma) [reply swarm/Dockerfile.1709]: I'm afraid yes. Without it an exception is generated:

```panic: Failed to load netapi32.dll: The specified module could not be found.

goroutine 1 [running]:
panic(0xc313a0, 0xc042326720)
	/usr/local/go/src/runtime/panic.go:500 +0x1af
syscall.(*LazyProc).mustFind(0xc04203d8c0)
	/usr/local/go/src/syscall/dll_windows.go:282 +0x71
syscall.(*LazyProc).Addr(0xc04203d8c0, 0x11523a0)
	/usr/local/go/src/syscall/dll_windows.go:289 +0x32
syscall.NetGetJoinInformation(0x0, 0xc0421e5be8, 0xc0421e5be4, 0x0, 0xc042068000)
	/usr/local/go/src/syscall/zsyscall_windows.go:1759 +0x38
os/user.isDomainJoined(0x665b4c, 0x3c11c20, 0x0)
	/usr/local/go/src/os/user/lookup_windows.go:21 +0x5a
os/user.lookupFullName(0xc04203b7c0, 0xc, 0xc04203b7b0, 0xd, 0xc0420c7b80, 0x1a, 0xd, 0xc0420c7b80, 0x1a, 0x0)
	/usr/local/go/src/os/user/lookup_windows.go:58 +0x2d
os/user.newUser(0xc042043cd0, 0xc04203b7a0, 0xc, 0xc0420c7b20, 0x16, 0xc04201c000, 0x0, 0x0)
	/usr/local/go/src/os/user/lookup_windows.go:88 +0x23f
os/user.current(0x0, 0x0, 0x0)
	/usr/local/go/src/os/user/lookup_windows.go:124 +0x1cc
os/user.Current(0xc04203b790, 0xc, 0xc04203b790)
	/usr/local/go/src/os/user/lookup.go:9 +0x29
github.com/docker/swarm/vendor/github.com/golang/glog.init.2()
	/Users/NishantTotla/work/src/github.com/docker/swarm/vendor/github.com/golang/glog/glog_file.go:63 +0x49
github.com/docker/swarm/vendor/github.com/golang/glog.init()
	/Users/NishantTotla/work/src/github.com/docker/swarm/vendor/github.com/golang/glog/glog_file.go:125 +0x1cd
github.com/docker/swarm/vendor/github.com/mesos/mesos-go/mesosutil.init()
	/Users/NishantTotla/work/src/github.com/docker/swarm/vendor/github.com/mesos/mesos-go/mesosutil/node.go:24 +0x49
github.com/docker/swarm/cluster/mesos/task.init()
	/Users/NishantTotla/work/src/github.com/docker/swarm/cluster/mesos/task/utils.go:17 +0x62
github.com/docker/swarm/cluster/mesos.init()
	/Users/NishantTotla/work/src/github.com/docker/swarm/cluster/mesos/utils.go:40 +0x5a
github.com/docker/swarm/cli.init()
	/Users/NishantTotla/work/src/github.com/docker/swarm/cli/manage.go:341 +0xcd
main.init()
	/Users/NishantTotla/work/src/github.com/docker/swarm/main.go:15 +0x49
panic: Failed to load netapi32.dll: The specified module could not be found.

goroutine 1 [running]:
```

I have too little knowledge about go see any other way here.

Comment (StefanScherer) [reply swarm/Dockerfile.1709]: Thanks for confirming it. 
Ok so swarm also retrieves some user information where golang wants to load this DLL. See https://github.com/golang/go/issues/21867

**Comment by StefanScherer** [comment]
LGTM

**Comment by StefanScherer** [comment]
I'm updating the provided image on Docker Hub to support 2016 + 1709 with a manifest list. A newer version of my rebase-docker-image tool is now able to rebase such golang binaries to have this netapi32.dll.

**Pull request merged**